### ISSUE TITLE
Keyboard Focus

### ISSUE THREAD

**Issue by EnsignPayton** [EnsignPayton/RealmListManager#20]
Early in development I disabled the default visual style for keyboard focus on visual elements. This needs to be re-added and customized so the user can navigate with keyboard only.

* Keyboard focus should give the same visual effect to buttons that hovering does.
* Location and Realmlist list controls should be navigable with tab and shift-tab, as well as up and down arrows. Ideally focus would not select items until Enter is pressed, but this is not a hard requirement. As long as it is possible to select another location from keyboard only, it is sufficient.
* There are no plans to offer screen reader support, so only interactive elements should be focusable.
